plugins/content_distribution/lib/api/KalturaCaptionDistributionInfo.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * @package plugins.contentDistribution
+ * @subpackage api.objects
+ *
+ */
+class KalturaCaptionDistributionInfo extends KalturaObject{
```

Comment:
In second thought, the content-distribution plugin today is not pending on caption-assets and we don't want it to.
You can pend facebook plugin on captions if you think it's needed, but adding captions already solved by other distribution providers such as you-tube without creating dependency.